### ISSUE TITLE
Colors and their transparency don't feel like mac

### ISSUE THREAD

**Issue by xidddekate** [vinceliuice/WhiteSur-gtk-theme#179]
have a look at this 
[https://drive.google.com/file/d/1IqCevxHsEDYk4K5Hx-ehl9Oo1mvO6XHi/view?usp=sharing](url) 

[https://drive.google.com/file/d/1MeDpY0ciK817WGT8BnwMuNyG6-4PIHGJ/view?usp=sharing](url)

I think that colors and their transparency can be tweaked to give a proper feel of mac to the end users. You may look at other macOS gnome themes in gnome-look website . Some have proper tranparency effect like that of mac.

Hope this suggestion makes theme more mac-ish

**Comment by rivanfebrian123** [comment]
Just reupload the images you provide to other developes (your link doesn't work, sorry)

![Screenshot from 2021-04-10 23-03-22](https://user-images.githubusercontent.com/6300910/114294456-02bd5a80-9ac9-11eb-8472-1bab2b5ecda5.png)

![abc](https://user-images.githubusercontent.com/6300910/114294462-081aa500-9ac9-11eb-82be-5a5a1b1aae53.png)


I don't have any Mac machines, so I don't know how the dark theme looks in a real Mac hardware. @vinceliuice you have a real MacBook?

**Comment by vinceliuice** [comment]
No I don't have mac, I only use linux and windows, gnome-shell is not easy to get a real mac like theme, because gnome don't support any blur effect

**Comment by CyberAzure** [comment]
I think it could be more transparent? I think this is possible.
And, the dark theme could be more gray, I think it's too dark.
Just search "MacOS Big Sur Dark Mode" on the internet and look up the images, you can get better what I'm trying to say.

**Comment by rivanfebrian123** [comment]
Yeah, and this too #214

**Comment by CoolAller** [comment]
> I think it could be more transparent? I think this is possible.

Yes, it's possible) Just change the "opacity" (transparency) of the image assets/menu.svg using inkscape to the one you need.
